llvm/test/CodeGen/AIE/GlobalISel/eliminate-duplicate-phi.mir

```diff
@@ -0,0 +1,132 @@
+    successors: %bb.2(0x80000000); %bb.2(100.00%)
+    %12:_(p0) = G_PHI %5:_(p0), %bb.1, %15:_(p0), %bb.2
+    %14:_(p0) = G_PHI %5:_(p0), %bb.1, %15:_(p0), %bb.2
+    %34:_(p0) = G_PHI %5:_(p0), %bb.1, %15:_(p0), %bb.2
+    %0:_(<32 x s8>) = G_LOAD %12:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+    %1:_(<32 x s8>) = G_LOAD %14:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+    %2:_(<32 x s8>) = G_LOAD %34:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+    G_BR %bb.2
+...
+
+---
+name:            duplicate_PHI_2_duplicate_elimination_2_unique
+legalized:       false
+tracksRegLiveness: true
+body:             |
+  ; CHECK-LABEL: name: duplicate_PHI_2_duplicate_elimination_2_unique
+  ; CHECK: bb.0:
+  ; CHECK-NEXT:   successors: %bb.1(0x80000000)
+  ; CHECK-NEXT:   liveins: $p1
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   [[C:%[0-9]+]]:_(p0) = G_CONSTANT i20 10
+  ; CHECK-NEXT:   [[C1:%[0-9]+]]:_(p0) = G_CONSTANT i20 0
+  ; CHECK-NEXT:   [[C2:%[0-9]+]]:_(p0) = G_CONSTANT i20 1000
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT: bb.1:
+  ; CHECK-NEXT:   successors: %bb.1(0x80000000)
+  ; CHECK-NEXT: {{  $}}
+  ; CHECK-NEXT:   [[PHI:%[0-9]+]]:_(p0) = G_PHI [[C]](p0), %bb.0, [[C1]](p0), %bb.1
+  ; CHECK-NEXT:   [[PHI1:%[0-9]+]]:_(p0) = G_PHI [[C]](p0), %bb.0, [[C1]](p0), %bb.1
+  ; CHECK-NEXT:   [[PHI2:%[0-9]+]]:_(p0) = G_PHI [[C]](p0), %bb.0, [[C2]](p0), %bb.1
+  ; CHECK-NEXT:   [[PHI3:%[0-9]+]]:_(p0) = G_PHI [[C]](p0), %bb.0, [[C1]](p0), %bb.1
+  ; CHECK-NEXT:   [[LOAD:%[0-9]+]]:_(<32 x s8>) = G_LOAD [[PHI]](p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+  ; CHECK-NEXT:   [[LOAD1:%[0-9]+]]:_(<32 x s8>) = G_LOAD [[PHI]](p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+  ; CHECK-NEXT:   [[LOAD2:%[0-9]+]]:_(<32 x s8>) = G_LOAD [[PHI]](p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+  ; CHECK-NEXT:   [[LOAD3:%[0-9]+]]:_(<32 x s8>) = G_LOAD [[PHI2]](p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+  ; CHECK-NEXT:   G_BR %bb.1
+  bb.1:
+    successors: %bb.2(0x80000000); %bb.2(100.00%)
+    liveins: $p1
+    %5:_(p0) = G_CONSTANT i20 10
+    %15:_(p0) = G_CONSTANT i20 0
+    %16:_(p0) = G_CONSTANT i20 1000
+
+  bb.2:
+    successors: %bb.2(0x80000000); %bb.2(100.00%)
+    %12:_(p0) = G_PHI %5:_(p0), %bb.1, %15:_(p0), %bb.2
+    %14:_(p0) = G_PHI %5:_(p0), %bb.1, %15:_(p0), %bb.2
+    %44:_(p0) = G_PHI %5:_(p0), %bb.1, %16:_(p0), %bb.2
+    %34:_(p0) = G_PHI %5:_(p0), %bb.1, %15:_(p0), %bb.2
+    %0:_(<32 x s8>) = G_LOAD %12:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+    %1:_(<32 x s8>) = G_LOAD %14:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+    %2:_(<32 x s8>) = G_LOAD %34:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
+    %3:_(<32 x s8>) = G_LOAD %44:_(p0) :: (load (<32 x s8>) from `ptr addrspace(5) null`, addrspace 5)
```

Comment:
Can you order the PHI nodes and the loads in the same order? Otherwise it's a bit confusing when reading :D